index.html

```diff
@@ -4,10 +4,10 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
```

Comment:
Put CSS Styles in the head section of the HTML file.

Reply:
Great job on implementing some of the requirements suggested by the previous reviewer, However, please you didn't make your comment polite enough to the student. You can do this by adding `please` or `kindly` before making your suggestions 😄